optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/measurement/ScoreDifferencePercentage.java

```diff
@@ -38,12 +38,12 @@
         }
         double[] percentageLevels = new double[baseLevels.length];
         for (int i = 0; i < baseLevels.length; i++) {
-            percentageLevels[i] = calculatePercentageLevel(baseLevels[i], valueLevels[i]);
+            percentageLevels[i] = calculateScoreDifferencePercentageLevel(baseLevels[i], valueLevels[i]);
         }
         return new ScoreDifferencePercentage(percentageLevels);
     }
 
-    private static double calculatePercentageLevel(double baseLevel, double valueLevel) {
+    private static double calculateScoreDifferencePercentageLevel(double baseLevel, double valueLevel) {
```

Comment:
I've made this method protected instead of private now. If you feel you every need to make methods protected or even public to write better or easier tests, feel free to do so in the PR. That's (almost) never a problem on implementation classes.

Note: Extracting methods up into interfaces or superclasses is a different matter. That depends on the case. Of course, you're still to propose PR's that do that, but I will be harder on letting such changes get in.